pkg/cluster/internal/providers/docker/images.go

```diff
@@ -35,21 +35,22 @@ func ensureNodeImages(logger log.Logger, status *cli.Status, cfg *config.Cluster
 	// pull each required image
 	for _, image := range common.RequiredNodeImages(cfg).List() {
 		// prints user friendly message
+		friendlyImageName := image
```

Comment:
I opted to keep the friendlyImageName around, but another solution to this is to remove the original logic that removed the sha256 portion. Does mean all the messages/logs would display the full SHA. What are folks' thoughts? I'm happy to adjust this in any way.